frodo.js

```diff
@@ -2,68 +2,65 @@
  * Extras Plugin
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.